labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -70,16 +59,18 @@ object ListManipulationExercise01 {
    * - ... whichever way you like 
    * 
    */
-  def sortList[T <% Ordered[T]](list: List[T]): List[T] = {
-    error("fix me")
-  }
+  def sortList[T <% Ordered[T]](list: List[T]): List[T] = { list.sorted }
 
   /**
    * Check whether a given element in a list exists, i.e. elementExists(List("a", "b", "c"), "b") = true
    * Again, easy to implement using built-in functionality, but also possible to implement in your own free-style way.
    */
   def elementExists[T](l: List[T], e: T): Boolean = {
-    error("fix me")
+    l match {
+      case x::tail if x == e => true
+      case x::tail => elementExists(tail, e)
```

Comment:
You could also try to combine the two cases that are so similar, such as 
`case x::tail => (x == e) || elementExists(tail, e)`. That only works well because `||` is short-circuiting, though.